docs/README.md

```diff
@@ -1,105 +1,411 @@
+
+### Step by step 
+
+>ℹ️ *To successfully perform this step by step, you may need previous knowledge of [Apollo Client](https://www.apollographql.com/docs/tutorial/introduction/?_ga=2.219865305.1929567151.1606434433-711418368.1606434433). Apollo Client is a library for JavaScript that works as a GraphQL client. It includes built-in features that allow declarative data fetching from a GraphQL server.*
+
+1. Open your React app in any code editor of your preference, and edit the `manifest.json` file, including the `vtex.geolocation-graphql-interface` app as a dependency.
+
+```json
+  "dependencies": {
+    "vtex.geolocation-graphql-interface": "0.x"
+  },
+```
+
+2. Using the terminal, access your app's directory and run the command below to install the [React Apollo](https://www.npmjs.com/package/react-apollo) npm library:
+
+```
+yarn add react-apollo
+```
+
+3. In your front-end app's repository, create a folder named `graphql` inside `react`.
+4. Save all the necessary queries your UI components rely on in `.graphql` files. 
+
+Take the [Location Search](https://github.com/vtex-apps/place-components/blob/37f42e94588e2cef897a7a0ff1e0df9e30ce2305/react/LocationSearch.tsx) component from the [Place Components](https://github.com/vtex-apps/place-components) app as an example. Notice that the [Location Search](https://github.com/vtex-apps/place-components/blob/37f42e94588e2cef897a7a0ff1e0df9e30ce2305/react/LocationSearch.tsx) component relies on the [`suggestAddresses.graphql`](https://github.com/vtex-apps/place-components/blob/master/react/graphql/suggestAddresses.graphql) query, defined as:
+
+```graphql
+query SuggestAddresses($searchTerm: String!) {
+  suggestAddresses(searchTerm: $searchTerm)
+    @context(provider: "vtex.geolocation-graphql-interface") {
+    description
+    mainText
+    mainTextMatchInterval {
+      offset
+      length
+    }
+    secondaryText
+    externalId
+  }
+}
+```
+
+>⚠️ You must use the `@context` [directive](https://www.apollographql.com/docs/apollo-server/schema/directives/) to prevent any issues related to ambiguity in the query definition, specifying which provider will be used to fetch the data for your component.
+
+5. In your front-end components, declared in the `/react` folder, define how your component will consume the data fetched from the Geolocation API using [Apollo Client](https://www.apollographql.com/docs/tutorial/introduction/?_ga=2.219865305.1929567151.1606434433-711418368.1606434433).
+
+>ℹ️ *For more information on how to develop React components using VTEX IO and GraphQL, check our documentation on [Developing components using VTEX IO and React - Fetching data](https://vtex.io/docs/getting-started/desenvolva-componentes-usando-vtex-io-e-react/6/).*
+
+## Schema description
```

Comment:
Are those schemas generated automatically? If yes, how can I do that for the future releases, in order to keep the documentation in sync?